components/automate-cli/cmd/chef-automate/iam.go

```diff
@@ -199,52 +198,18 @@ func runRestoreDefaultAdminAccessAdminCmd(cmd *cobra.Command, args []string) err
 		}
 	}
 
-	// grant access to admins team if needed
-	resp, err := apiClient.PoliciesClient().GetPolicyVersion(ctx, &iam_req.GetPolicyVersionReq{})
+	foundAdminsTeaminV2AdminPolicy, err := adminmgmt.UpdateV2AdminsPolicyIfNeeded(ctx,
+		apiClient, iamCmdFlags.dryRun)
 	if err != nil {
-		return status.Wrap(err, status.APIError, "Failed to verify IAM version")
+		return err
 	}
 
-	writer.Titlef("Checking IAM %s policies for admin policy with admins team.\n", display(resp.Version))
-
-	switch resp.Version.Major {
-	case iam_common.Version_V1:
-		foundV1AdminPolicy, createdNewV1Policy, err := adminmgmt.UpdateV1AdminsPolicyIfNeeded(ctx,
-			apiClient, iamCmdFlags.dryRun)
-		if err != nil {
-			return err
-		}
-
-		if foundV1AdminPolicy {
-			writer.Skipped("Found admin policy that contains the admins team")
-		} else {
-			// Note: (tc) This should never happen currently since we currently don't support
-			// editing policies but adding for future-proofing against the functionality.
-			// Note: (sr) PurgeSubjectFromPolicies can alter policies -- when a user or a
-			// team is removed; so, this could be more realistic than we think.
-			writer.Successf("Found default admins team policy but it did not contain "+
-				"the admins team subject (%s). Added admins team to default admin policy.",
-				authz_constants.LocalAdminsTeamSubject)
-		}
-		if createdNewV1Policy {
-			writer.Success("Created new admins policy")
-		}
-	case iam_common.Version_V2:
-		foundAdminsTeaminV2AdminPolicy, err := adminmgmt.UpdateV2AdminsPolicyIfNeeded(ctx,
-			apiClient, iamCmdFlags.dryRun)
-		if err != nil {
-			return err
-		}
-
-		if !foundAdminsTeaminV2AdminPolicy {
-			writer.Success("Added local team: admins to Chef-managed policy: Admin")
-		}
-
-		writer.Skipped("Found local team: admins in Chef-managed policy: Admin")
-	default:
-		// do nothing
+	if !foundAdminsTeaminV2AdminPolicy {
+		writer.Success("Added local team: admins to Chef-managed policy: Admin")
 	}
 
+	writer.Skipped("Found local team: admins in Chef-managed policy: Admin")
```

Comment:
While you are in here how about make these flow better (and correct name):
```suggestion
	if !foundAdminsTeaminV2AdminPolicy {
		writer.Success("Added local 'admins' team to Chef-managed 'Administrator' policy")
	}

	writer.Skipped("Found local 'admins' team in Chef-managed 'Administrator' policy")
```